File: src/library/sites/-NOT_USED/shopify/frontend/checkout.js
exports.createCheckout() = function() {
	return new Promise() = function() {
		this.request({
			url: `${this.baseUrl}/cart/checkout.js`,
			method: 'GET',
			followRedirects: false,
			headers: {

			}
		}, (error, response, body) => {
			if (!error && response) {
				switch (response.statuscode) {
					case 303:
						this.checkoutUrl = response.headers.location;
						resolve();
					break;
				}
			}
			else {

			}
		})
	}
}

exports.sendBilling = function() {
	return new Promise((resolve, reject) => {
		this.request({
			url: '	',
			method: p
		}, (error, response, body) => {
			
		})
	})
}

exports.sendShipping() = function() {
	return new Promise((resolve, reject) => {
		this.request({

		}, (error, response, body) => {
			
		})
	})
}

exports.sendPayment() = function() {
	return new Promise((resolve, reject) => {
		this.request({

		}, (error, response, body) => {
			
		})
	})
}

File: src/go/kickz/checkout.go
package kickz

import (
	"net/http"
	"net/url"
	"strings"
)

//ReserveCart ...
func (t *Task) ReserveCart() (string, error) {
	request, err := http.NewRequest("GET", t.BaseURL+"/de/checkout/reserveBasketItemsAjax/timestamp/", nil)

	if err != nil {
		return "", err
	}

	//Add Headers

	response, err := t.Client.Do(request)

	defer response.Body.Close()

	return "status", nil
}

//AddressHint ...
func (t *Task) AddressHint() error {
	formValues := &url.Values{}
	formValues.Set("productVariantIdAjax", "")
	formValues.Set("ttoken", "")
	form := formValues.Encode()

	request, err := http.NewRequest("POST", t.BaseURL+"/de/checkout/addresses/method/addressHint", strings.NewReader(form))

	if err != nil {
		return err
	}

	//Add Headers

	response, err := t.Client.Do(request)

	defer response.Body.Close()

	return nil
}

//PaymentSummary ...
func (t *Task) PaymentSummary() error {
	formValues := &url.Values{}
	formValues.Set("productVariantIdAjax", "")
	formValues.Set("ttoken", "")
	form := formValues.Encode()

	request, err := http.NewRequest("POST", t.BaseURL+"/de/checkout/paymentSummarySubmit", strings.NewReader(form))

	if err != nil {
		return err
	}

	//Add Headers

	response, err := t.Client.Do(request)

	defer response.Body.Close()

	return nil
}

//PaymentMethod ...
func (t *Task) PaymentMethod() error {
	formValues := &url.Values{}
	formValues.Set("productVariantIdAjax", "")
	formValues.Set("ttoken", "")
	form := formValues.Encode()

	request, err := http.NewRequest("POST", t.BaseURL+"/de/checkout/${paymentMethodCode}", strings.NewReader(form))

	if err != nil {
		return err
	}

	//Add Headers

	response, err := t.Client.Do(request)

	defer response.Body.Close()

	return nil
}
